### PR TITLE
Notifications into monorepo

### PR DESCRIPTION
**Pull request by akramtexas**
- The goal of the work (for which this PR is being submitted) is to bring in the "support-notifications" Golang package into the EdgeX monorepo.

### CODE REVIEW

File: cmd/support-notifications/main.go [hunk 1]
```diff
@@ -85,7 +84,7 @@ func main() {
 		return
 	}
 
-	r := notifications.LoadRestRoutes()
+	//r := notifications.LoadRestRoutes()
```
Comment (tsconn23) [inline cmd/support-notifications/main.go]: This line commented out intentionally. It's redundant with content in the startHttpServer() function. I should have removed the line before commit. Perhaps @akramtexas can remove it.

**Comment by akramtexas** [comment]
`Related Blackbox tests continue to pass 100%.`

**Comment by akramtexas** [comment]
👍

File: cmd/support-notifications/main.go (hunk 1 above)
Comment (akramtexas) [reply cmd/support-notifications/main.go]: Good point @tsconn23 (Done, earlier. At this time, adding a comment here for acknowledgment and closure).

File: support/notifications/transmission.go [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+
+func transmissionByEndHandler(w http.ResponseWriter, r *http.Request) {
+
+	if r.Body != nil {
+		defer r.Body.Close()
+	}
+
+	vars := mux.Vars(r)
+	end, err := strconv.ParseInt(vars["end"], 10, 64)
+	// Problem converting start
```
Comment (drasko) [inline support/notifications/transmission.go]: Confusing comment.

**Pull request merged**